ansible/roles/test/tasks/everflow_testbed/testcase_8.yml

```diff
@@ -0,0 +1,76 @@
+# Test case 8 - Policer enforced DSCP value/mask test
+
+- set_fact:
+    policer_name: TEST_POLICER
+    policer_session_name: TEST_POLICER_SESSION
+    dscp_table_name: EVERFLOW_DSCP
+
+- name: Create route with next hop {{ dst_port_1 }}.
+  shell: vtysh -e "conf t" -e "ip route {{ session_prefix_1 }} {{ neighbor_info_1['addr'] }}"
+  become: yes
+
+- block:
+  - name: Create a policer
+    shell: |
+      redis-cli -n 4 hmset "POLICER|{{policer_name}}" "meter_type" "packets" "mode" "sr_tcm" "cir" "100" "cbs" "100" "red_packet_action" "drop"
+    become: yes
+
+  - name: Create a policer enforced mirror session
+    shell: |
+      config mirror_session add {{policer_session_name}} {{session_src_ip}} {{session_dst_ip}} {{session_dscp}} {{session_ttl}} --policer {{policer_name}}
```

Comment:
Hi @stcheng . It seems that on public master and 201811 images this command is not supported (see below). Is sonic-utilities submodule pointer going to be updated soon?

Log:
```
"cmd": "config mirror_session add TEST_POLICER_SESSION 1.1.1.1 2.2.2.2 8 1 --policer TEST_POLICER"
"stderr": "Error: no such option: --policer",
```

Reply:
@mykolaf I'll update the sonic-utilities pointer very soon